pkg/pods/pod.go

```diff
@@ -39,7 +39,7 @@ const DEFAULT_PATH = "/data/pods"
 const DefaultP2Exec = "/usr/local/bin/p2-exec"
 
 var DefaultFinishExec = []string{"/bin/true"} // type must match preparerconfig
-var DefaultLogExec = []string{"svlogd", "-tt", "./main"}
+var DefaultLogExec = []string{"chpst", "-unobody", "svlogd", "-tt", "./main"}
```

Comment:
Can we use `p2-exec` instead? I've been meaning to kill the `chpst` dependency from this project.